devel/apache-arrow/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 
 github.setup        apache arrow 3.0.0 apache-arrow-
-revision            6
+revision            4
```

Comment:
You are decreasing the revision here which is not allowed.

this, and a bunch if other changes in the PR tells me you have not properly based your changes on the current version in master. Please go back fix this to make sure the diff here is only the minimal changes need to fix whatever the conflict is you refer to.

Reply:
Yep, I started working on this refactor before your improvements were added and obviously missed these. The latest push is up-to-date.